README.md

```diff
@@ -2,264 +2,65 @@
-
-See [releases.md](/doc/releases.md) for more information on versioning.
-
-### Data Format
-
-The importer is capable of performing certain functions that streamline its use with Kubevirt.  It automatically decompresses **gzip** and **xz** files, and un-tar's **tar** archives. Also, **qcow2** images are converted into a raw image files needed by Kubevirt, resulting in the final file being a simple _.img_ file.
-
-Supported file formats are:
-
-- .tar
-- .gz
-- .xz
-- .img
-- .iso
-- .qcow2
-
-## Deploying CDI
-
-### Assumptions
-- A running Kubernetes cluster with roles and role bindings implementing security necesary for the CDI controller to watch PVCs and pods across all namespaces.
-- A storage class and provisioner.
-- An HTTP or S3 file server hosting VM images
-
-- An optional "golden" namespace acting as the image registry. The `default` namespace is fine for tire kicking.
-
-### Download CDI
-
-`$ go get kubevirt.io/containerized-data-importer`
-
-*Or*
-
-`$ git clone https://github.com/kubevirt/containerized-data-importer.git $GOPATH/src/kubevirt.io/containerized-data-importer`
+Deploying the CDI controller is straight forward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
 
-*Or download only the yamls:*
+  ```
+  $ VERSION=<cdi version>
+  $ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-controller.yaml
+  ```
 
-```shell
-$ mkdir cdi-manifests && cd cdi-manifests
-$ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/kubevirt-centric-readme/manifests/example/golden-pvc.yaml
-$ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/kubevirt-centric-readme/manifests/example/endpoint-secret.yaml
-```
+If you're running OpenShift, make sure you enable privileged containers
 
-### Deploy CDI from a release
+  ```
+  $ oc adm policy add-scc-to-user privileged -z cdi-sa
+  ```
 
-Deploying the CDI controller is straight forward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
+## Use it
 
-1. Ensure that the cdi-sa service account has proper authority to run privileged containers, typically in a kube environment this is true by default.
-   If you are running an openshift variation of kubernetes you may need to enable privileged containers in the security context:
+Create a DataVolume (Kubevirt Custom Object) which will provide a PVC with a Cirros disk image
 
 ```
-$ oc adm policy add-scc-to-user privileged -z cdi-sa
+$ kubectl create -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/release-v1.5/manifests/example/datavolume.yaml
```

Comment:
do we want the main workflow to be based on datavolumes ?

Reply:
I assumed we wanted that as the primary use case, do you disagree?  Or are you suggesting we omit the direct PVC use case?

Reply:
what i mean is that since datavolumes are feature gated, maybe they shouldnt be the primary use case shown

Reply:
Data volumes are feature gated in kubevirt, not in CDI.